src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const populationElem = document.body.querySelectorAll('.population');
+const population = Array.from(populationElem).map((elem) => {
+  return parseInt(elem.textContent.replace(',', ''), 10);
+});
+const total = population.reduce((accum, current) => accum + current);
+const average = total / population.length;
```

Comment:
Ensure that the population array is not empty before dividing by its length to calculate the average. If the array is empty, this will result in a division by zero error. Consider adding a check to handle this case.